src/client/dfs/dfs.c

```diff
@@ -3084,6 +3088,10 @@ dfs_open_stat(dfs_t *dfs, dfs_obj_t *parent, const char *name, mode_t mode,
 			stbuf->st_ctim.tv_sec = entry.ctime;
 		}
 		*_obj = obj;
+	} else if (rc == EEXIST) {
```

Comment:
this is definitely wrong.. eexist is a failure in open with O_CREAT so we cannot make it a success case and return the object!